### PR TITLE
chore(deps): update non-esm dependencies

### PR DESCRIPTION
**Pull request by perrin4869**
Updated all dependencies that didn't convert to pure-esm.
Since `serverless` >= 3.2 support native ESM, I'm thinking of making a `major` release supporting `node >= 14.17` and `serverless >= 3.2`, and convert this module to pure ESM. @theBenForce What do you think?

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install dependencies
-        uses: bahmutov/npm-install@v1
```
Comment (theBenForce) [inline .github/workflows/testing.yml:26]: Nice catch!

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (theBenForce) [inline .npmrc:1]: Does this prevent yarn from creating a `yarn.lock`?

**Review by theBenForce — APPROVED** [review]
You may want to change the commit message to prefix `feat:` and add a `BREAKING CHANGE:` to the footer so that it bumps to a new major version.

Version increment triggers: https://semantic-release.gitbook.io/semantic-release/support/faq#how-can-i-change-the-type-of-commits-that-trigger-a-release

**Comment by perrin4869** [comment]
Cool! I'll merge this now, do the rest of the changes in a separate PR!

File: .npmrc (hunk 2 above)
Comment (theBenForce) [reply .npmrc:1]: Yes, but you want to use the lock file in CI environments to ensure you're installing the same version of dependencies as you've been developing against.

**Pull request merged**